screens/HomeScreen.tsx

```diff
@@ -223,6 +233,28 @@ const HomeScreen = ({
     return tokensSorted;
   }, [balances.slpTokens, tokensById]);
 
+  const blacklistTokenData: WalletSection["data"] = useMemo(() => {
```

Comment:
Extra complexity here.  RIght now it's computing the blacklistToken data, then listening to changes to this, to compute the `blacklistSection`.  
Since this first `memo` value is only used to directly trigger and modify the other `memo` value, it makes sense to combine the code into a single memo.  This way the related logic stays together, less looking around.    The primary goal of the hook paradigm is colocating related code by hook/effect blocks instead of lifecycle events.
For example, that may be done like

```javascript
const blacklistSection:  WalletSection = useMemo(() => {
  const filteredTokens = tokenData.filter(data => tokenBlacklist.include(data.tokenId);
   return {
    title: 'Hidden Tokens',
   data: filteredTokens
  };
}, [tokenData, tokenBlacklist])
```

Then same for `sectionSLP`

Reply:
these changes caused the SectionList to crash. I was also getting Type errors when trying to refactor accordingly.

Seeing as how the functionality already works and there are no performance issues, I do not feel it is worth spending the extra cycles to refactor for little to no gain.

Reply:
I wrote psuedo-code for this, so will require a small change.
It's worth doing because learning React + the hook paradigm is super important going forward.  If you won't make this change, I will.